library/bz2.po

```diff
@@ -127,47 +158,65 @@ msgid ""
 "can equivalently be given as ``'rb'``, ``'wb'``, ``'xb'`` and ``'ab'`` "
 "respectively."
 msgstr ""
+"L'argument *mode* peut être soit ``'r'``  pour lire (par défaut), ``'w'`` "
+"pour écraser, ``'x'`` pour créer exclusivement, ou ``'a'`` pour ajouter. Ils "
+"peuvent également être écrits respectivement comme ``'rb'``, ``'wb'``, "
+"``'xb'`` et ``'ab'``."
 
 #: ../Doc/library/bz2.rst:81
 msgid ""
 "If *filename* is a file object (rather than an actual file name), a mode of "
 "``'w'`` does not truncate the file, and is instead equivalent to ``'a'``."
 msgstr ""
+"Si *filename* est un objet fichier (plutôt que le nom de fichier réel), le "
+"mode ``'w'`` ne tronque pas le fichier, mais équivaut à ``'a'``."
 
 #: ../Doc/library/bz2.rst:84
 msgid "The *buffering* argument is ignored. Its use is deprecated."
-msgstr ""
+msgstr "L'argument *buffering* est ignoré. Son usage est déprécié."
 
 #: ../Doc/library/bz2.rst:86
 msgid ""
 "If *mode* is ``'w'`` or ``'a'``, *compresslevel* can be an integer between "
 "``1`` and ``9`` specifying the level of compression: ``1`` produces the "
 "least compression, and ``9`` (default) produces the most compression."
 msgstr ""
+"Si *mode* est ``'w'`` ou ``'a'``, *compresslevel* peut être un entier entre "
+"``1`` et ``9`` spécifiant le niveau de compression : ``1`` produit une "
+"compression la moins forte, et ``9`` (par défaut) la compression la plus "
+"forte."
 
 #: ../Doc/library/bz2.rst:90
 msgid ""
 "If *mode* is ``'r'``, the input file may be the concatenation of multiple "
 "compressed streams."
 msgstr ""
+"Si *mode* est ``'r'``, le fichier d'entrée peut être la concaténation de "
+"plusieurs flux compressés."
 
 #: ../Doc/library/bz2.rst:93
 msgid ""
 ":class:`BZ2File` provides all of the members specified by the :class:`io."
 "BufferedIOBase`, except for :meth:`detach` and :meth:`truncate`. Iteration "
 "and the :keyword:`with` statement are supported."
 msgstr ""
+":class:`BZ2File` fournit tous les membres spécifiés par la classe :class:`io."
+"BufferedIOBase`, excepté les méthodes :meth:`detach` et :meth:`truncate`. "
+"L'itération et l'instruction :keyword:`with` sont supportées."
```

Comment:
```suggestion
"L'itération et l'instruction :keyword:`with` sont pris en charge."
```

Reply:
Étant donné que les termes "instructions" et "itération" sont deux féminins, à moins que je ne me trompe, je ne valide pas cette correction :

"prises en charge"

Reply:
Effectivement.